scripts/gee

```diff
@@ -993,15 +993,17 @@ function _install_tools() {
       | sudo /usr/bin/apt-key --keyring "${KRFILE}" add -
     echo "deb [arch=amd64 signed-by=${KRFILE}] https://cli.github.com/packages stable main" \
       | sudo /usr/bin/tee /etc/apt/sources.list.d/githubcli-archive.list
-    sudo /usr/bin/apt update && sudo /usr/bin/apt install gh
+    sudo /usr/bin/apt update || /bin/true  # ignore errors associated with stale repos.
```

Comment:
not objecting, but for my own understanding: why do you specify the full path for something like /bin/true?

(also apt, to be frank, or tee - eg, full path is a double edged sword: works without PATH, yay, but things do move around, so eg, even across ubuntu versions I've seen the path of pretty basic shell commands change - was recently bit by one)

Reply:
It's a good security hygiene habit.  One of common attack on scripts is to insert a malicious program into the path and trick a script to run it with elevated privileges.

Doesn't matter for gee, which doesn't run with any special privileges, but I've just gotten into the habit of trying to do things the secure way.